src/lib/dlt_user.c

```diff
@@ -4708,6 +4698,11 @@ int dlt_start_threads()
         dlt_log(LOG_CRIT, "Can't create housekeeper thread!\n");
         return -1;
     }
+    if (pthread_setname_np(dlt_housekeeperthread_handle, "dlt_housekeeper"))
+    {
+        dlt_log(LOG_CRIT, "Can't rename housekeeper thread!\n");
+        return -1;
```

Comment:
I wonder if it should return error in case the thread name could not be renamed

Reply:
An error may occur if the name does not fit to what the OS can handle. This is probably mostly an issue when running the code the first time on another platform. I think it's nice to handle this properly.  On the other hand, you are right, it not a critical error though, the code still works without renaming.

Reply:
I also see that some OS, even some Linux system, does not support `pthread_setname_np()`, so we need to consider that case too, otherwise it will fail to compile.
We internally have a solution to check existence of the call, let me raise a separate PR for it and you can change on top of that. How does that sound? :)

Reply:
FYI: https://github.com/GENIVI/dlt-daemon/pull/329

Reply:
Thanks for pointing this out. It looks like `pthread_setname_np()` was previously already in use. I have updated the change accordingly.